lein-finagle-clojure/src/leiningen/finagle_clojure.clj

```diff
@@ -1,16 +1,35 @@
 (ns leiningen.finagle-clojure
   (:require [leiningen.javac]
             [robert.hooke]
-            [clojure.java.io :as io]))
+            [clojure.java.io :as io]
+            [leiningen.core.classpath :as classpath]
+            [cemerick.pomegranate :as pom])
+  (:import (java.io File)
+           (java.net URL)))
 
-(defn- thrift-files
+(defn- find-thrift-files
   [project-root source-path]
   (->> source-path
     (io/file project-root)
     (file-seq)
     (filter #(and (.isFile %) (.endsWith (.getName %) ".thrift")))
     (map #(.getPath %))))
 
+(defn- scrape-includes
+  "Returns the filenames of thrift files this code tries to include."
+  [thrift-code]
+  (map second (re-seq #"include \"(.+\.thrift)\"" thrift-code)))
+
+(defn- copy-file-to-target-temp-dir
+  "Copies a thrift file we depend on into target/thrift-include"
+  [project-root ^URL thrift-url filename]
+  (let [target-dir (File. project-root "target/thrift-include")
```

Comment:
You repeat the constant `target/thrift-include` a couple times - better to refactor it to a single `def`